ci/test-miri.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+source ci/rust-version.sh nightly
+
+rustup component add miri --toolchain "$rust_nightly"
+
+# miri is very slow; so only run very few of selective tests!
+cargo "+${rust_nightly}" miri test -p solana-program -- hash:: account_info::
```

Comment:
does the `#[cfg_attr(miri)]` work for this purpose? (I'm not quite understand how it select tests. try to steal your brain 😂)

Reply:
that needs to add `#[cfg_attr(miri, ignore)]` to almost all tests...